standalone/pherry/README.md

```diff
@@ -0,0 +1,3 @@
+Download GeoLite-City.mmdb database at https://drive.google.com/file/d/1UDKHuZ2KQSaDvy34LMKlUPKgToZPoNUk/view?usp=sharing
+
+Placing it at the root of pherry folder (./standalone/pherry/GeoLite-City.mmdb).
```

Comment:
> @kvinwang Where do you think is the best place to put the file? I don't think under `pherry/` is a good idea.

If we can drop the file into our repo, we can see how others do: https://github.com/Phala-Network/phala-blockchain/search?q=include_bytes
So, I think `standalone/pherry/data/GeoLite-City.mmdb` is OK.

Reply:
I was worried that the file was too big (70MB). Considering this file might be updated frequently (Maxmind updates this database every week), it is likely to finally exceed the storage limit for Github Repo.

Reply:
> Considering this file might be updated frequently (Maxmind updates this database every week)

In this case，we should load the data at run time rather than `include_bytes!`.